WordPress/UITestsFoundation/Screens/NotificationsScreen.swift

```diff
@@ -111,7 +111,9 @@ public class NotificationsScreen: ScreenObject {
 
 
     public func likeComment() -> Self {
+        XCTAssertTrue(likeCommentButton.label.hasSuffix(.commentNotLikedLabel))
         likeCommentButton.tap()
+        XCTAssertTrue(likeCommentButton.label.hasSuffix(.commentLikedLabel))
```

Comment:
I have several thoughts on this addition.

1. These assertions are part of an "action" method, which is generally not expected to contain assertions. IMO it'll be more consistent to move the first line to a new `verifyCommentNotLiked`, and move the last line to existing `verifyCommentLiked`.
2. These assertions are really fast. The flakiness might be reduced by adding these two because they still take some time, which increases the stability. But, for the better measure, WDYT of waiting for an element with `commentLikedLabel` to appear after the tap instead?

Reply:
i like the suggestion, especially to confirm that the element really is updated. 

i removed the asserts before and after the tap and moved the `.commentLikedLabel` one to `verifyCommentLiked()`. since we can't do a `waitForExistence()` directly on the label value, i did a while loop to check if it's updated and the assert after that. this is updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/21420/commits/b83ec8609519da5e79d9edf4b9d09e7812ae3796